logger/sentry.go

```diff
@@ -96,6 +97,15 @@ func (sl *SentryLogger) send(level sentry.Level, ctx *LogContext) {
 			scope.SetContext("data", ctx.Data)
 		}
 
+		// attempt to populate tags from Data
+		tags := convertMapAnyToString(ctx.Data)
+		for k, v := range tags {
+			// Sentry tag keys/values are limited to 32 and 200 chars respectively
+			if len(k) <= 32 && len(v) <= 200 {
+				scope.SetTag(k, v)
```

Comment:
What about something like to pass along whatever we can?

```suggestion
	maxK := 32
	maxV := 200

	if len(k) < maxK {
		maxK = len(k)
	}

	if len(v) < maxV {
		maxV = len(v)
	}

	scope.SetTag(k[:maxK], v[:maxV])
```

Reply:
cool yeah works for me